### DISCUSSION TITLE
Load GLTF react three-fiber

### DISCUSSION THREAD

**Discussion by HildLuger** [vercel/community#2833] [Help] [answered]
### Summary

My project is working perfectly on my local server, but whem I deploy it to vercel I got a error informing that it could not load my gltf file.

The file is in the public folder, anyone knows what it could be?

  
  

What it could be?

### Example

https://wondrous-moonbeam-5097b0.netlify.app/

### Steps to Reproduce

Heres is the loader inside my component. The file is in plublic folder.

import { useLoader, useThree, useFrame } from '@react-three/fiber';
import { GLTFLoader } from 'three/examples/jsm/loaders/GLTFLoader.js';

export default function Skull() {
  const { scene } = useLoader(GLTFLoader, './public/skull2.gltf');
  
   It runs smothly on my local server but in vercel i got this error:
  
  index-fdf4541d.js:3523     GET https://wondrous-moonbeam-5097b0.netlify.app/public/skull2.gltf 404
load @ index-fdf4541d.js:3523
load @ index-fdf4541d.js:3602
(anonymous) @ index-fdf4541d.js:3565
(anonymous) @ index-fdf4541d.js:3565
(anonymous) @ index-fdf4541d.js:3565
n2 @ index-fdf4541d.js:3565
W4 @ index-fdf4541d.js:3565
vS @ index-fdf4541d.js:3565
kV @ index-fdf4541d.js:3618
S0 @ index-fdf4541d.js:3551
uw @ index-fdf4541d.js:3553
sw @ index-fdf4541d.js:3553
tI @ index-fdf4541d.js:3553
qd @ index-fdf4541d.js:3553
ew @ index-fdf4541d.js:3553
A @ index-fdf4541d.js:3539
K @ index-fdf4541d.js:3539
index-fdf4541d.js:3553 Error: Could not load ./public/skull2.gltf: fetch for "https://wondrous-moonbeam-5097b0.netlify.app/public/skull2.gltf" responded with 404: )
    at index-fdf4541d.js:3565:25131
    at Object.a [as onError] (index-fdf4541d.js:3602:15299)
    at index-fdf4541d.js:3523:193110
R0 @ index-fdf4541d.js:3553
MM.b.componentDidCatch._.callback @ index-fdf4541d.js:3553
WS @ index-fdf4541d.js:3551
ZM @ index-fdf4541d.js:3553
KM @ index-fdf4541d.js:3553
JL @ index-fdf4541d.js:3553
iI @ index-fdf4541d.js:3553
Aa @ index-fdf4541d.js:3553
ew @ index-fdf4541d.js:3553
A @ index-fdf4541d.js:3539
K @ index-fdf4541d.js:3539
index-fdf4541d.js:40 Error: Could not load ./public/skull2.gltf: fetch for "https://wondrous-moonbeam-5097b0.netlify.app/public/skull2.gltf" responded with 404: )
    at index-fdf4541d.js:3565:25131
    at Object.a [as onError] (index-fdf4541d.js:3602:15299)
    at index-fdf4541d.js:3523:193110
ly @ index-fdf4541d.js:40
SC.t.callback @ index-fdf4541d.js:40
$w @ index-fdf4541d.js:38
gE @ index-fdf4541d.js:40
UC @ index-fdf4541d.js:40
xN @ index-fdf4541d.js:40
TN @ index-fdf4541d.js:40
Va @ index-fdf4541d.js:40
kC @ index-fdf4541d.js:40
A @ index-fdf4541d.js:25
K @ index-fdf4541d.js:25
index-fdf4541d.js:3565 Uncaught Error: Could not load ./public/skull2.gltf: fetch for "https://wondrous-moonbeam-5097b0.netlify.app/public/skull2.gltf" responded with 404: )
    at index-fdf4541d.js:3565:25131
    at Object.a [as onError] (index-fdf4541d.js:3602:15299)
    at index-fdf4541d.js:3523:193110
(anonymous) @ index-fdf4541d.js:3565
a @ index-fdf4541d.js:3602
(anonymous) @ index-fdf4541d.js:3523

**Comment by amyegan** [comment, marked as answer]
Hi @HildLuger. It looks like you might be trying to deploy a Netlify project. If that's the case, you should reach out in their support forum.

Otherwise, the error seems accurate as https://wondrous-moonbeam-5097b0.netlify.app/public/skull2.gltf does respond with a `404`
